src/Standards/BestIt/CodeSniffer/Helper/DocTagHelper.php

```diff
@@ -69,29 +61,35 @@ public function __construct(array $commentStartToken, File $file, int $stackPtr,
      */
     private function getCommentStartToken(): array
     {
-        return $this->commentStartToken;
+        return $this->token;
     }
 
     /**
      * Loads the tag content for the given tag position.
      *
      * @param int $tagPosition The position of the tag.
+     * @param int $iteratedPosition
      *
      * @return array The content tokens of the tag.
      */
-    private function loadTagContentTokens(int $tagPosition): array
+    private function loadTagContentTokens(int $tagPosition, int &$iteratedPosition): array
     {
         $contents = [];
-        $nextOrClosingPos = $this->file->findNext([T_DOC_COMMENT_CLOSE_TAG, T_DOC_COMMENT_TAG], $tagPosition + 1);
-        $positionsTillEnd = TokenHelper::findNextAll(
-            $this->file->getBaseFile(),
-            [T_DOC_COMMENT_STRING],
-            $tagPosition,
-            $nextOrClosingPos
-        );
-
-        foreach ($positionsTillEnd as $position) {
-            $contents[] = $this->tokens[$position];
+        $myColumn = $this->tokens[$tagPosition]['column'];
```

Comment:
This recursion was needed for the fix. This helper did not render the child contents correctly.